.github/CODEOWNERS

```diff
@@ -143,6 +143,7 @@
 /stackpulse/                             @torqio/rnd support@torq.io
 /stardog/                                @snowell support@stardog.com
 /statsig/                                @marcos-statsig support@statsig.com
+/steadybit/                              @bripkens support@steadybit.com
```

Comment:
Please include @DataDog/marketplace-review to the codeowners file

Reply:
done